pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/migration/AngioMigrationRun.java

```diff
@@ -1,84 +1,27 @@
 package org.broadinstitute.ddp.model.migration;
```

Comment:
Code under migration package is not in use any more. I think we don't need to invest time to refactor it.
It was written and used to migrated some studies from different system to pepper.

Reply:
Should we delete this code then?